src/client/Microsoft.Identity.Client/Platforms/Android/Broker/BrokerRequest.cs

```diff
@@ -121,6 +121,12 @@ private static BrokerRequest FromAuthenticationParameters(AuthenticationRequestP
                 br.Claims = _authenticationRequestParameters.Claims;
             }
 
+            if (_authenticationRequestParameters.RequestContext.ServiceBundle.Config.ClientCapabilities?.Any() == true)
```

Comment:
How about just using `ClaimsAndClientCapabilities` here? i.e. `br.Claims = _authenticationRequestParameters.ClaimsAndClientCapabilities`

Reply:
It would override the existing claims. This is additive i.e. just adds to any existing.